src/commonMain/kotlin/baaahs/sim/FakeNetwork.kt

```diff
@@ -47,6 +47,10 @@ class FakeNetwork(
             return FakeUdpSocket(serverPort)
         }
 
+        override fun mdns(): Network.Mdns {
+            TODO("not implemented")
```

Comment:
We'll want to do some trivial implementation here so it keeps working in the simulator. (... and invoke it from `Brain.kt` somehwere.